lingua_franca/lang/parse_en.py

```diff
@@ -1459,20 +1355,20 @@ def isFractional_en(input_str, short_scale=True):
 
     fracts = {"whole": 1, "half": 2, "halve": 2, "quarter": 4}
     if short_scale:
-        for num in _SHORT_ORDINAL_STRING_EN:
+        for num in _SHORT_ORDINAL_EN:
             if num > 2:
-                fracts[_SHORT_ORDINAL_STRING_EN[num]] = num
+                fracts[_SHORT_ORDINAL_EN[num]] = num
     else:
-        for num in _LONG_ORDINAL_STRING_EN:
+        for num in _LONG_ORDINAL_EN:
             if num > 2:
-                fracts[_LONG_ORDINAL_STRING_EN[num]] = num
+                fracts[_LONG_ORDINAL_EN[num]] = num
 
     if input_str.lower() in fracts:
         return 1.0 / fracts[input_str.lower()]
     return False
 
 
-def extract_numbers_en(text, short_scale=True, ordinals=False):
+def extract_numbers_en(text, short_scale=None, ordinals=False):
```

Comment:
i will need to backtrack to remember why, i remember making this change when adding short_scale=None to extract_numbers to be passed down to the individual language methods, it was necessary for some reason

i think None basically meant to use the default, when passing True or False down the pipeline it was an explicit user choice, i will need to go over the code again to give more details on why this was needed

Reply:
oh thats it, extract_numbers would either always pass True or False if a default was provided, not allowing a per language default, so a check for None is needed, either in individual methods or in parent extract_numbers method before calling individual languages, doing it in individual_methods keeps language specific code in language specific files

Reply:
Wondering if it will be cleaner to handle this up the chain with something like:
```Python
def extract_numbers(text, short_scale=None, ordinals=False, lang=None):
    if short_scale is None:
        args = (text, ordinals)
    else:
        args = (text, short_scale, ordinals)
    lang_code = get_primary_lang_code(lang)
    if lang_code == "en":
        return extract_numbers_en(*args)
```

Reply:
i like that, will make changes